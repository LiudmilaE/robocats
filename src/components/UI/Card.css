.Card-box {
    background-color: #F4E6ED;
    min-width: 250px;
    padding: 0.5rem;
    margin: 1.5rem;
    border-radius: 0.5rem;
    -webkit-box-shadow: 4px 13px 54px 0px rgba(0,0,0,0.75);
       -moz-box-shadow: 4px 13px 54px 0px rgba(0,0,0,0.75);
            box-shadow: 4px 13px 54px 0px rgba(0,0,0,0.75);
}