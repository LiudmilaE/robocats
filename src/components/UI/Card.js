import React from 'react';
import './Card.css';

const Card = ({title, name}) => (
  <section className="Card-box">
    <img src={`https://robohash.org/${title}.png?size=200x200&set=set4`}
      alt={title}/>
    <h3>{title}</h3>
    <p>Owner - {name}</p>
  </section>
);

export default Card;