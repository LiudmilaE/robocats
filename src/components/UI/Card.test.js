import React from 'react';
import { shallow } from 'enzyme';
import Card from './Card';

describe("<Card/>", () => {
    it('renders and match the snapshot', () => {
        expect(shallow(<Card />)).toMatchSnapshot();
    });
});