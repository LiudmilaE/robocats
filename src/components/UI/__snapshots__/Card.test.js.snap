// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Card/> renders and match the snapshot 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Card />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <img
          alt={undefined}
          src="https://robohash.org/undefined.png?size=200x200&set=set4"
/>,
        <h3 />,
        <p>
          Owner - 
</p>,
      ],
      "className": "Card-box",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "alt": undefined,
          "src": "https://robohash.org/undefined.png?size=200x200&set=set4",
        },
        "ref": null,
        "rendered": null,
        "type": "img",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": undefined,
        },
        "ref": null,
        "rendered": null,
        "type": "h3",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            "Owner - ",
            undefined,
          ],
        },
        "ref": null,
        "rendered": Array [
          "Owner - ",
          undefined,
        ],
        "type": "p",
      },
    ],
    "type": "section",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <img
            alt={undefined}
            src="https://robohash.org/undefined.png?size=200x200&set=set4"
/>,
          <h3 />,
          <p>
            Owner - 
</p>,
        ],
        "className": "Card-box",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "alt": undefined,
            "src": "https://robohash.org/undefined.png?size=200x200&set=set4",
          },
          "ref": null,
          "rendered": null,
          "type": "img",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": undefined,
          },
          "ref": null,
          "rendered": null,
          "type": "h3",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              "Owner - ",
              undefined,
            ],
          },
          "ref": null,
          "rendered": Array [
            "Owner - ",
            undefined,
          ],
          "type": "p",
        },
      ],
      "type": "section",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
}
`;
